src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4872,4 +4872,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureTuples" xml:space="preserve">
     <value>tuples</value>
   </data>
+  <data name="IDS_FeatureOutVar" xml:space="preserve">
+    <value>out var</value>
+  </data>
```

Comment:
Sure, I'll change in the next PR.

---
In reply to: [63741830](https://github.com/dotnet/roslyn/pull/11393#discussion_r63741830) [](ancestors = 63741830)